javascript/javascript3/week1/giphySearch.css

```diff
@@ -0,0 +1,37 @@
+body {
+  background-color: rgb(238, 246, 243);
+  font-size: larger;
+  margin: auto;
+}
+
+* {
+  margin: 0.5rem;
+  padding-left: 0.1rem;
+  box-sizing: border-box;
+}
+
+input[type="text"],
+input[type="number"] {
+  min-width: fit-content;
+  padding: 0.25rem;
+}
+
+input[type="button"] {
+  background-color: aquamarine;
+  border-radius: 5px;
+  padding: 0.75rem;
+}
+
+ul {
+  list-style-type: none;
+}
+
+img {
+  width: 100%;
```

Comment:
Try to style with classNames/ids instead of element names. That would really matter in the future.